challenges/1-media-alunos/c/spacexjedi/README.MD

```diff
@@ -0,0 +1,21 @@
+# Submissão de Exercicio
+
+**Exercicio:** 1 - Media de Alunos
+
+**Nickname:** spacexjedi
+
+**Nível Técnico:** - Estudante
+
+**Twitter**: none
+
+**Dificuldade de Resolução:** - Baixa
+
+**Comentários:** Nesse desafio eu achei que ... (Opcional)
```

Comment:
Retire as linhas do readme.md que você não for utilizar!

Reply:
editei